webpack/ForemanTasks/Components/TasksTable/TasksTablePage.js

```diff
@@ -63,7 +63,10 @@ const TasksTablePage = ({
         toastNotifications="foreman-tasks-cancel"
         toolbarButtons={
           <React.Fragment>
-            {props.status === STATUS.PENDING && <Spinner size="lg" loading />}
+            <Button onClick={() => props.reloadPage(url, props.parentTaskID)}>
+              {__('Refresh Data')}
```

Comment:
can you add a refresh icon too? e.g: https://fontawesome.com/v4.7.0/icon/refresh
or maybe it can be just a refresh icon without the text.. saw that in several places of our app